packages/amplication-server/.env

```diff
@@ -47,3 +47,8 @@ KAFKA_BROKERS="["localhost:9092"]"
 KAFKA_CLIENT_ID='server-queue-client'
 KAFKA_GROUP_ID = 'main-server-group'
 
+BASE_BUILDS_FOLDER=
+
+# Novu:
+NOVU_API_KEY=
```

Comment:
is this key public ?

Reply:
we have a development value and production value for it. I am not sure what you mean by 'public', but without this key there is no communication between the server and the client

Reply:
This value is a secret key used to communicate with the notifications API. Better to keep it in your secret management store for the backend.